src/WinRT.Runtime2/InteropServices/RestrictedErrorInfo.cs

```diff
@@ -75,6 +75,17 @@ public static void SetErrorInfo(Exception exception)
         // TODO
     }
 
+    /// <summary>
+    /// Attaches the error info stored by the <c>IRestrictedErrorInfo</c> infrastructure to the input exception.
+    /// </summary>
+    /// <param name="exception">The input <see cref="Exception"/> instance to attach the error info to.</param>
+    /// <returns>The input <see cref="Exception"/> instance with attached error info.</returns>
+    public static Exception AttachErrorInfo(Exception exception)
```

Comment:
Is this only meant to be used by projections? I'm thinking if so we might want to mark this one and the other ones in `RestrictedErrorInfo` that are not for general use as private implementation detail methods (or move them).

Reply:
Will address in upcoming PR.